docs/source/receptorctl/receptorctl_status.rst

```diff
@@ -0,0 +1,171 @@
+--------------------------
+Receptorctl status command
+--------------------------
+
+.. contents::
+   :local:
+
+``receptorctl status`` shows the Receptor network status
+
+The command syntax is: ``receptorctl --socket=<socket_path> status [--json]``
+
+``socket_path`` is the control socket address to connect to Receptor.
+   It defaults to Unix socket.
+   Use tcp:// for TCP socket.
+   The corresponding environment variable is ``RECEPTORCTL_SOCKET``
+
+``ss --listening --processes --unix 'src = unix:<socket_path>'``
+Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
+
+``ps -fp $(pidof receptor)``
+``lsof -p <<pid>``
+
+``--json`` will format the output as JSON.
+   The default is human-readable.
+   Using this argument allows output to be machine consumable.  For example, piping into ``jq``
+
+The output is divided into sections listed below.  Field values may be listed in their own section.
+Columns are the actual JSON node values.
+
+^^^^^^^^^^^^
+Node Section
+^^^^^^^^^^^^
+
+.. list-table:: Node Section
+      :header-rows: 1
+      :widths: auto
+
+      * - Column
+        - Description
+      * - ``."NodeID"``
+        - Node Identifier
+      * - ``."SystemCPUCount"``
+        - Number of CPUs on node
+      * - ``.SystemMemoryMiB"``
+        - Memory (MiB)
+      * - ``."Version"``
+        - Receptor version
+
+^^^^^^^^^^^^^^^^^^^
+Connections Section
+^^^^^^^^^^^^^^^^^^^
+
+.. list-table:: Connections Section
+    :header-rows: 1
+    :widths: auto
+
+    * - Column
+      - Description
+    * - ``."Connections"``
+      - Connections section
+    * - ``."Connections"."Cost"``
+      - Cost to reach NodeID
```

Comment:
Is there another word here? What do we mean by "cost"? Is it latency in terms of milliseconds?

In general better to not use the same word as the thing you're trying to describe.